3/assets/3.3-finished-code.rs

```diff
@@ -118,7 +118,12 @@ decl_module! {
 
             <balances::Module<T>>::make_transfer(&sender, &owner, kitty_price)?;
 
-            Self::transfer_from(owner.clone(), sender.clone(), kitty_id)?;
+            Self::transfer_from(owner.clone(), sender.clone(), kitty_id)
+                .expect("`owner` is shown to own the kitty; \
+                `owner` must have greater than 0 kitties, so transfer cannot cause underflow; \
```

Comment:
```suggestion
```
Having a kitten (line before) also proofs they have more than `0` kitties. I'd drop this line from the proof as it is already stated.